e2e/keywords/node.resource

```diff
@@ -0,0 +1,61 @@
+*** Settings ***
+Documentation       Kubernetes cluster nodes related keywords resources
+
+Library             ../libs/keywords/volume_keywords.py
+
+
+*** Variables ***
+${POD_EVICTION_TIME_OUT}=       300
+
+
+*** Keywords ***
+Get the index of the cluster nodes
+    ${cluster_node_index}    ${node_index_with_replica}    ${node_index_without_replica} =    get_cluster_node_index
+    ...    ${volume_name}
+    ...    ${request_replica_count}
+    Set Test Variable    ${cluster_node_index}
+    Set Test Variable    ${node_index_with_replica}
+    Set Test Variable    ${node_index_without_replica}
+
+Power off node ${node_index}
+    ${target_node_index} =    Evaluate    ${node_index}-1
+    Run keyword And Continue On Failure
+    ...    Wait Until Keyword Succeeds
```

Comment:
I think this place can indeed remove the retry mechanism, because there is already a waiting mechanism in the function. https://github.com/longhorn/longhorn-tests/blob/e427e2fc9aa5bc251559558e5eab0db678161f8d/e2e/libs/node/operations/aws.py#L43-L50
So I will only keep the retry mechanism when checking the workloads state.